frontend/packages/console-app/src/components/pdb/modals/index.ts

```diff
@@ -0,0 +1,6 @@
+import { DeletePDBModalProps } from './DeletePDBModal';
+
+export const deletePDBModal = (props: DeletePDBModalProps) =>
+  import('./DeletePDBModal' /* webpackChunkName: "shared-modals" */).then((m) =>
+    m.deletePDBModal(props),
+  );
```

Comment:
Any reason you're lazy loading the modal caller? 🤔 

I'm unable I can think of a reason to do this given that the rest of the code isn't lazy loaded.

Reply:
Thought it was a conversation to lazy load modals. The user could visit the rest of the code lazy loaded without clicking the modal. So the modal chunk is not included in the rest of the code bundle.